frontend/server/src/Controllers/Course.php

```diff
@@ -1637,6 +1637,27 @@ public static function apiArbitrateRequest(\OmegaUp\Request $r): array {
             ])
         );
 
+        if (!is_null($targetIdentity->user_id)) {
+            \OmegaUp\DAO\Notifications::create(
+                new \OmegaUp\DAO\VO\Notifications([
+                    'user_id' => $targetIdentity->user_id,
+                    'contents' =>  json_encode(
+                        [
+                            'type' => $request->accepted ? \OmegaUp\DAO\Notifications::COURSE_REGISTRATION_ACCEPTED : \OmegaUp\DAO\Notifications::COURSE_REGISTRATION_REJECTED,
```

Comment:
nit:

```suggestion
                            'type' => (
                                $request->accepted ?
                                \OmegaUp\DAO\Notifications::COURSE_REGISTRATION_ACCEPTED :
                                \OmegaUp\DAO\Notifications::COURSE_REGISTRATION_REJECTED
                            ),
```

así la línea no es excesivamente larga. Igual en L1648.